examples/liveness/README.md

```diff
@@ -0,0 +1,27 @@
+This example shows two types of pod health checks: HTTP checks and containter execution checks. This [guide](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/walkthrough/k8s201.md) has more information on health checks.
+
+To show the health check is actually working, first create the pods:
+```
+# cluster/kubectl.sh create -f exec-liveness.yaml
+# cluster/kbuectl.sh create -f http-liveness.yaml
+```
+
+Check the status of the pods once they are created:
+```
+# cluster/kubectl.sh get pods
+POD             IP           CONTAINER(S)   IMAGE(S)                            HOST                                     LABELS          STATUS    CREATED     MESSAGE
+liveness-exec   10.244.3.7                                                      kubernetes-minion-f08h/130.211.122.180   test=liveness   Running   3 seconds   
+                             liveness       gcr.io/google_containers/busybox                                                             Running   2 seconds   
+liveness-http   10.244.0.8                                                      kubernetes-minion-0bks/104.197.10.10     test=liveness   Running   3 seconds   
+                             liveness       gcr.io/google_containers/liveness                                                            Running   2 seconds   
+```
+
+Check the status half a minute later, you will see the termination messages:
```

Comment:
You should mention that the liveness probe failed, and the container has been killed and recreated (hence the termination message).

Another way of showing this is using `/cluster/kubectl.sh describe pod liveness-exec`. You would be able to see the restart count increment as time goes by.
```
Name:				liveness-exec
Image(s):			gcr.io/google_containers/busybox
Host:				127.0.0.1/127.0.0.1
Labels:				test=liveness
Status:				Running
Replication Controllers:	<none>
Containers:
  liveness:
    Image:		gcr.io/google_containers/busybox
    State:		Running
      Started:		Thu, 14 May 2015 11:28:06 -0700
    Ready:		True
    Restart Count:	4
```
You would also see the killing and creating events at the bottom of the output.